Documentation/migration-scripts/migrate-org-roles.sh

```
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# DB connection details
+DB_NAME="test"
+DB_USER="postgres"
+DB_HOST="localhost"
+PGPASSWORD="Test@123"
+DB_PORT="5432"
```

Shellcheck (0.11.0)

[warning] DB_PORT appears unused. Verify use (or export if used externally).

(SC2034)